content/artifacts/artifacts-finder/plugin.js

```diff
@@ -0,0 +1,140 @@
+// Artifacts Finder
+// This plugin can help auto prospect and find artifacts.
+// Simple is Power!
+
```

Comment:
Maybe mention start finding runs on a 10 minute timer in the background, and the only way to stop finding is to close the plugin.

Reply:
When start finding, the button change to "Cancel Finding", you can stop finding by click button, or close the button. The different is click close button can keep the finding log.